tests/foreman/cli/test_role.py

```diff
@@ -188,7 +188,7 @@ def make_role_with_permissions(self):
 
     @tier1
     @upgrade
-    @pytest.mark.parametrize('per_page', **parametrized([1, 5, 20]))
```

Comment:
this method isn't doing anything useful for this parameter. It just results in ID's of 0, 1, 2 instead of 1, 5, 20, which are pretty meaningless.